src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/GraphWorkspaceDAO.scala

```diff
@@ -11,7 +11,7 @@ import scala.language.implicitConversions
 
 class GraphWorkspaceDAO extends WorkspaceDAO with GraphDAO {
 
-  override def save(workspace: Workspace, txn: RawlsTransaction): Workspace = txn withGraph { db =>
+  override def save(workspace: Workspace, txn: RawlsTransaction): WorkspaceContext = txn withGraph { db =>
```

Comment:
at least some return Unit.  but the idea is that WorkspaceContext is consistently exactly what the DAO loads and saves if this PR is accepted.